week0/homework/person.js

```diff
@@ -0,0 +1,29 @@
+
+ exports.person=class person 
+ {
+ constructor(name,age){
+    this.name=name;
+    this.age=age;
+  };
+addPhone (num){
+    return this.num=num;
+}
+call (){
+    if(this.num !=null){
+     console.log("Calling "+ this.name+ " at : " +this.num);
```

Comment:
Remember, template strings in ES5/6 are much better to use for string construction: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals